timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -424,7 +424,7 @@ def fit(
                 f"Provided tuning data set with {len(tuning_data)} rows, {tuning_data.num_items} items. "
                 f"Average time series length is {len(tuning_data) / tuning_data.num_items:.1f}."
             )
-        logger.info(f"Training artifacts will be saved to: {Path(self.path).resolve()}")
```

Comment:
Isn't Autogluon not only saving models, but also saving other training artifacts such as predictor and learner states?

Reply:
My goal here was to 
1. Remove a duplicated print statement (the path is printed twice)
2. Make the output of `TimeSeriesPredictor` similar to the output of `TabularPredictor`:
```
Beginning AutoGluon training ...
AutoGluon will save models to "AutogluonModels/ag-20221108_130039/"
```
How about `AutoGluon will save models and other training artifacts to {...}`?